Jenkinsfile

```diff
@@ -0,0 +1,153 @@
+pipeline {
+    agent {
+        label 'python'
+    }
+
+    environment {
+        docker_repo = "indigodatacloud/app"
```

Comment:
@gwarf note that this variable should contain the appropriate dockerhub repository

Reply:
@orviz gret!
So the correct repository is the following: https://hub.docker.com/r/egifoundation/cloud-info-provider/
Which user jenkins will use? Who should I allow to push containers?